scripts/wasm-rebuild/docker-scripts/isolate_tests.py

```diff
@@ -36,7 +36,7 @@ def extract_test_cases(path):
 
 def extract_and_write(f, path):
     if f.endswith('.sol'):
-        with open(path, 'r') as _f:
+        with open(path, 'r', encoding='utf8') as _f:
```

Comment:
Note: this script is unmaintained and uses Python 2. In Python 2 `open()` does not have the `encoding` argument so this will make it raise an exception. I added it anyway because there are other places in this file that already do this (so the script must be broken already) and it silences pylint.